src/Hedgehog/Gen.fs

```diff
@@ -69,20 +72,26 @@ module Gen =
     let map (f : 'a -> 'b) (g : Gen<'a>) : Gen<'b> =
         mapTree (Tree.map f) g
 
+#if !FABLE_COMPILER
```

Comment:
I guess `CompiledName` breaks fable compiler, right?

Reply:
JavaScript doesn't support overloading for functions, just like F#.

Reply:
I wonder if it's be possible to (somehow) feed those .fs files to the Fable compiler and strip (on the fly) those attributes.

Reply:
As far as I know Fable compiler understands CompiledName attributes (I think I saw them in Fable source code) but it cannot generate two Methods with the same name.

Reply:
Yep, I just checked. Fable compiler understands CompiledName attribute. There is an JsInteropTest file where it is tested: https://github.com/fable-compiler/Fable/blob/c600b6e030b78bd9605749f1b8c72e8b2c73fdde/tests/Main/JsInteropTests.fs#L29